ophasebase/models.py

```diff
@@ -152,6 +152,8 @@ class Meta:
     name = models.CharField(max_length=100, verbose_name=_('Name'))
     description_template = models.CharField(max_length=50, verbose_name=_('Beschreibung in Template'))
     lang = models.CharField(max_length=5, verbose_name=_('Sprachcode'), default="de")
+    details_template = models.CharField(max_length=50, verbose_name=_('Template für Stundenplan View'))
```

Comment:
The slug would then be used to lookup the corresponding category in the website model (enforce unique).